### ISSUE TITLE
BTT Pico to USB

### ISSUE THREAD

**Issue by MimmoLagonigro** [bigtreetech/SKR-Pico#26]
hi guys I wanted to connect the BTT Pico directly to my linux pc via USB port instead of the classic UART method.
Is it possible to use the USB port that we use to install Klipper for data transmission as well?
or do I need to create a UART to USB adapter and if so, what is the arrangement and order of the cables?
thanks in advance

**Comment by mysy200** [comment]
Yes, I use it on my two Voron 0.2R1. Connect to other Btt Pi usb port
